test/scripts/base-host-check.sh

```diff
@@ -140,6 +140,21 @@ check_hostname() {
     fi
 }
 
+check_services_enabled() {
+    echo "📗 Checking enabled services"
+
+    services_expected=$(jq -rc '.blueprint.customizations.services.enabled[]' "${config}")
+
+    echo "$services_expected" | while read -r service_expected; do
+        state=$(systemctl is-enabled "${service_expected}")
+        if [[ "${state}" == "enabled" ]]; then
+            echo "Service was enabled service=${service_expected} state=${state}"
+        else
+            echo "❌ Service was not enabled service=${service_expected} state=${state}"
+        fi
+    done
```

Comment:
Shouldn't the check `exit 1` in case there are any failures? :thinking:

Reply:
Done.